cloudapp/src/app/main/main.component.html

```diff
@@ -22,6 +22,7 @@ <h3>Select record(s) to print:</h3>
                                       [value]="user.id"
                                       (change)="onUserToggled($event)"
                                       [checked]="user.checked"
+                                      [disabled]="user.addresses.length==0"
```

Comment:
This is just to prevent selecting a row with no address

Reply:
Sounds plausible 👍